Readme.md

```diff
@@ -3,7 +3,7 @@
 ## Purpose of this repository
 
 This Git repository documents, configures and automates the management and setup of the Sovereign Cloud Stack hardware environment.
-This environment is build in the context of the VP18 project workpackage and in running at a colocation of [JH Computers](https://jh-computers.de/).
+This environment is build in the context of the VP18 project workpackage and in running at a colocation of [ScaleUp Technologies](https://scaleuptech.com).
```

Comment:
… was built in the context of the SCS project (work package VP18) … and is today/now running at the BER3 datacenter of …